stable/restore/files/nuorestore

```diff
@@ -10,9 +10,47 @@
 #   --source    - source to restore from - can be a backupset or a URL
 #   --auto      - automatically start the restore operation ( true | false )
 
+[ "$NUODB_DEBUG" = "verbose" ] && set -X
+
 : ${NUODB_BACKUP_KEY:=/nuodb/nuobackup}
 : ${NUODB_RESTORE_REQUEST_PREFIX:=/nuodb/nuosm}
 
+LOGFILE=${NUODB_LOGDIR:=/var/log/nuodb}/nuorestore.log
+
+#=======================================
+# function - log messages
+#
+function log() {
+  if [ -n "$NUODB_DEBUG" ]; then
+    echo "nuorestore $@" | tee -a $LOGFILE
+   #  curl -k -X POST -E /etc/nuodb/keys/nuocmd.pem -H "Content-type: application/json" $ap_protocol://$NUOADMIN_AP0:8888/api/1/diagnostics/log -d "{\"message\": \"nuorestore $@\"}"
```

Comment:
Can we remove the commented out code? Also, we now have `nuocmd log message`, so we would not have to use `curl`.